packages/react-native/ReactCommon/React-rncore.podspec

```diff
@@ -37,6 +37,7 @@ Pod::Spec.new do |s|
   s.source                 = source
   s.source_files           = "dummyFile.cpp"
   s.pod_target_xcconfig = { "USE_HEADERMAP" => "YES",
+                            "HEADER_SEARCH_PATHS" => header_search_paths.join(' '),
```

Comment:
React-rncore does not have any `header_search_path` variable set up.
This is making all the build on iOS failing.
Also, this dependency is not depending on anything else, so we can probably remove this line completely.

Reply:
I'm sorry, it appears my patch didn't get applied correctly when I made this commit so it's missing content! I'll fix it shortly.